.env.example

```diff
@@ -0,0 +1,13 @@
+# Network Configuration
+# Set to 'true' for testnet (Sepolia), 'false' for mainnet
+NEXT_PUBLIC_IS_TESTNET=true
```

Comment:
Didn't notice it the last time. Curious about the `NEXT_` prefix

Reply:
This is something NextJS design to explicitly inform developers that these variables will be bundled into the static frontend, and visible to all users on browsers.